setup.cfg

```diff
@@ -22,7 +22,7 @@ classifiers =
 [options]
 include_package_data = True
 install_requires =
-    Django>=3.0.0
+    Django>=3.1,<3.2
```

Comment:
We don't use pyup bot on this project ?

Reply:
Nope.
Not yet (I just created #75 )

Reply:
@lunika Pyup is now set up on this project.
Django version has been pinned to 3.1 and future upgrades will be handled by Pyup.